### ISSUE TITLE
[Enhancement] Prompt user if incident query returns a large dataset

### ISSUE THREAD

**Issue by giranm** [giranm/pd-live-react#52]
## Summary
PagerDuty's [REST API pagination](https://developer.pagerduty.com/docs/ZG9jOjExMDI5NTU4-pagination) currently supports up to a maximum of 10,000 records returned.  
When querying for incidents (particularly over +1000 results), this can impact the user experience of:

- Fetching incident notes due to current API limitations (e.g. initial notes are fetched sequentially)
- Bulk update of incidents (acknowledge, resolve, etc) can be slow to manage

To advise the user that a large incident dataset will be returned, PagerDuty Live should have the capability of checking what the query would return, and then prompting the user to continue if it exceeds a predefined limit of incidents.

#### Example API Query
GET: `https://api.pagerduty.com/incidents?limit=1&total=true`

```json
{
    "incidents": [
        {
            "incident_number": 18082,
            "title": "Warning: High Response Time Detected",
            "description": "Warning: High Response Time Detected"
            ...
        }
    ],
    "limit": 1,
    "offset": 0,
    "total": 409,
    "more": true
}
```

The `total` field can then be used to prompt the user if the entire dataset should be downloaded.

**Comment by giranm** [comment]
Another route to consider is redirecting the user to the main Global Search page with the same query parameters.

`https://subdomain.pagerduty.com/search/?startDate=2022-03-01&endDate=2022-03-11&searchString=error&searchType[]=incident&service[]=PABCXYZ&sortBy=relevance&team[]=PXYZABC`

NB - `searchString` must be populated otherwise no incidents will be returned. Need to identify if a wildcard can be used.

**Comment by giranm** [comment]
Closing issue since this was addressed in #127.
